docs/HowToUseTaskMonitor.md

```diff
@@ -0,0 +1,79 @@
+# How to use Task-monitor
+
+Task monitor is kernel thread which monitor the tasks/pthreads.
+It checks if registered tasks/pthreads are alive.
+
+## Contents
+- [Enable task-monitor](#enable-task-monitor)
+- [Use task-monitor](#use-task-monitor)
+
+
+## Enable task-monitor
```

Comment:
Please change task-monitor to upper case.

Reply:
modified